hardhat.config.ts

```diff
@@ -66,6 +68,11 @@ const config: HardhatUserConfig = {
       verbose: true,
     },
   },
+  etherscan: {
+    apiKey: {
+      goerli: ETHERSCAN_API_KEY!,
```

Comment:
@rahulrumalla we need to add this to this repo's secrets in Github (and if it doesn't read the variable, figure out why)

Reply:
Got it. Added now